apps/vr-tests-react-components/src/stories/Positioning.stories.tsx

```diff
@@ -442,6 +442,29 @@ const VirtualElement = () => {
   );
 };
 
+const ResetTarget = () => {
+  const [virtualElement, setVirtualElement] = React.useState<PositioningVirtualElement | null>(null);
+  const { containerRef, targetRef } = usePositioning({
+    position: 'below',
+    align: 'end',
+    target: virtualElement,
+  });
+
+  React.useEffect(() => {
+    if (virtualElement) {
+      setVirtualElement(null);
+    }
+  }, [virtualElement]);
+
+  return (
+    <>
+      <button ref={setVirtualElement}>Virtual Target</button>
```

Comment:
nit: maybe have the virtual target on top of the target, that way it should be easier to see that the screenshot is incorrect beacause the popover on the virtual target will be covering the target